sacro/views.py

```diff
@@ -32,11 +36,20 @@ def __post_init__(self):
     def content_urls(self):
         urls = {}
         for output, data in self.items():
-            params = {"path": str(self.path), "file": data["output"]}
+            params = {"path": str(self.path), "name": output}
             urls[output] = reverse_with_params(params, "contents")
 
         return urls
 
+    def get_file_path(self, name):
+        """Return absolute path to output file"""
+        path = Path(self[name]["output"])
+
+        if path.is_absolute():
```

Comment:
If I remove this if-statement and make this function always return `return self.path.parent / path` then the unit test  `test_contents_absolute` passes, but I think it should fail? I'm confident that the code path is being executed because returning a random string causes an error.

Reply:
Hmm, maybe pathlib.Path is doing some kind of magic if you add an absolute path to a absolute path?

Reply:
Yup 
```python
>>> from pathlib import Path 
>>> a = Path(r"C:\test")
>>> b = Path(r"C:\Users\wavy\sacro\tests")
>>> a / b
WindowsPath('C:/Users/wavy/sacro/tests')
>>>```

Reply:
Same behaviour in linux.

So, this is feature of pathlib it seems, and I hates it.

But technically, it means that code branch is not needed, although the test is.  But I think I prefer the clarity of the code branch?

Also, if ACRO changes its defaults to use relative paths, then absolute paths would be an error?